proposals/lip-0062.md

```diff
@@ -39,35 +39,72 @@ Introduction of another hash function to the Lisk protocol was suggested in [LIP
 
 ## Specification
 
-The signature for a binary message `message` and a secret key `sk` is generated by `signMessage(sk, tag, networkIdentifier, message)` as defined below. `tag` must be the correct message tag for `message` as defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md), and `networkIdentifier` the correct network identifier for the chain. The resulting signature `signature` in combination with the message `message` and the matching public key `pk` is verified by `verifyMessageSig(pk, tag, networkIdentifier, message, signature)`.
+### Signing and Verifying with Ed25519
+
+The Ed25519 signature for a binary message `message` and a secret key `sk` is generated by `signMessage(sk, tag, chainID, message)` as defined below. `tag` must be the correct message tag for `message` as defined in [LIP 0037](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0037.md), and `chainID` the correct chain ID for the chain. The resulting signature `signature` in combination with the message `message` and the matching public key `pk` is verified by `verifyMessageSig(pk, tag, chainID, message, signature)`.
 
 ```python
-signMessage(sk, tag, networkIdentifier, message):
-    taggedMessage = tag || networkIdentifier || message
-    hashedMessage = hash(taggedMessage)
+def signMessage(sk: bytes, tag: bytes, chainID: bytes, message: bytes) -> bytes:
```

Comment:
For clarity, should we rename `signMessage` to `signEd25519` (or something similar) because we also have `signBLS`.

Reply:
I agree, but I will ask since this could impact other LIPs/code